src/test/java/com/egov/core-services/userDetails/qa/userDetails.yaml

```diff
@@ -2,3 +2,11 @@
 superUserCounterEmployee:
   userName: EMP-107-000177
   password: 79m7be9
+
+#UserName and Password Super User
+superUser:
+  userName: EMP-1013-000335
+  password: TestUser@1234
+  tenantId: pb.jalandhar
```

Comment:
i think we have discussed to split tenantId into stateId and cityId instead calling like tenantId: pb.jalandhar